docs/source/visualisation/experiment_tracking.md

```diff
@@ -1,49 +1,66 @@
 # Experiment tracking in Kedro-Viz
 
-Experiment tracking is a way to record all information you need to compare machine-learning experiments and recreate those experiments. All experiment data (parameters, metrics, models, plots and other dataset types) can be searched using a timestamp.
+Experiment tracking is the process of saving all the metadata related to an experiment each time you run it. It enables you to compare different runs of a machine-learning model as part of the experimentation process.
 
-> View a detailed [Kedro-Viz experiment tracking demo](https://demo.kedro.org/experiment-tracking).
+The metadata you store may include:
 
-In the past, Kedro had limited support for experiment tracking functionality. For example, it was possible to version parameters as part of your codebase with a version control system like `git` and [snapshot models, datasets and plots with Kedro’s dataset versioning capabilities](../data/data_catalog.md#version-datasets-and-ml-models). However, Kedro needed a way to log metrics and capture all this logged data as a timestamped run of an experiment. It also needed a way for users to visualise, discover and compare this data.
+* Scripts used for running the experiment
+* Environment configuration files
+* Versions of the data used for training and evaluation
+* Evaluation metrics
+* Model weights
+* Plots and other visualisations
 
-We started adding in the missing pieces from [Kedro-Viz `>=4.1.1`](https://github.com/kedro-org/kedro-viz/releases/tag/v4.1.1). Now, when experiment tracking is enabled in your Kedro project, you will be able to access, edit and [compare your experiments](#access-run-data-and-compare-runs) and additionally [track how your metrics change over time](#view-and-compare-metrics-data).
+Early versions of Kedro supported parameter versioning (as part of your codebase with a version control system like `git`) and Kedro’s dataset versioning capabilities enabled you to [snapshot models, datasets and plots](../data/data_catalog.md#version-datasets-and-ml-models).
 
-![](../meta/images/experiment-tracking_demo_small.gif)
+More recent versions of Kedro and Kedro-Viz introduce metadata capture, visualisation, discovery and comparison, enabling you to access, edit and [compare your experiments](#access-run-data-and-compare-runs) and additionally [track how your metrics change over time](#view-and-compare-metrics-data).
+
+Take a look at the [Kedro-Viz experiment tracking demo](https://demo.kedro.org/experiment-tracking) for further information.
+
+![](../meta/images/experiment-tracking_demo.gif)
```

Comment:
Thanks @tynandebold  -- I'll need someone to help me with the GIF creation as it's not something I can do smoothly (I have made those new images). 

Shall we make a separate ticket to update the images to GIFs in the near future rather than hold back on the updates to make them in this PR? 

Or go for it and get the new graphics in before we make the release? I can share my code (it's on my repo https://github.com/stichbury/kedro-projects/tree/main/spaceflights_with_experiment_tracking/kedro-experiment-tracking-tutorial) if you can help me make the GIFs.

Reply:
I think it's fine to use the demo project for making the GIF for that first image.

Why can't you make the GIFs smoothly? Is it because you don't have metrics plot setup in your local project? Or is it a tech issue?

Reply:
It's user error. I assume I need to screen record myself operating viz, and I'm going to be hopelessly flailing the cursor around and not showing off the product to the best of itself. How do you do it? Let me catch you tomorrow morning before/after standup and get some tips.

Reply:
Deal :)